spec/spec_helper.rb

```diff
@@ -16,6 +16,23 @@
 require_relative "spec_help/fixture"
 require_relative "spec_help/test_logger"
 require_relative "spec_help/toxiproxy_config"
+require_relative "spec_help/http_test_server"
+require_relative "spec_help/port_prober"
+
+module HttpHelpers
+  def with_real_http_connections
+    original_cassette = VCR.current_cassette
+    VCR.eject_cassette if original_cassette
```

Comment:
It should be possible to do this, as [`VCR.eject_cassette` returns `nil` when no cassette has been inserted](https://www.rubydoc.info/github/vcr/vcr/VCR#eject_cassette-instance_method):

```suggestion
    original_cassette = VCR.eject_cassette
```